src/mesido/esdl/profile_parser.py

```diff
@@ -100,10 +100,16 @@ def read_profiles(
             zip(esdl_asset_id_to_name_map.values(), esdl_asset_id_to_name_map.keys())
         )
 
+        # Define variable containing potential error information
+        asset_potential_errors = {"heat_demand.power": []}
```

Comment:
maybe instead of the list with the error messages, make a nested dict, with the heatdemand name as they key and then the message as the value. Leaves more flexibility towards future uses in case we only want to extract the asset names.

Reply:
Do you mean:
asset_potential_errors = { 
       "heat_demand.power": {
                       heat_demand_name: "error message"
        }
}

Reply:
Indeed :)

Reply:
Nice I will update the code

Reply:
Code has been updated